brainglobe_atlasapi/atlas_generation/atlas_scripts/demba_dev_mouse.py

```diff
@@ -0,0 +1,345 @@
+        )
+        volume_resolution = 25
+    else:
+        raise ValueError(f"Unknown modality: {modality}")
+    return reference_path, annotation_path, volume_resolution
+
+
+def retrieve_reference_and_annotation(
+    download_dir_path, age, resolution, modality
+):
+    """
+    Retrieve the desired reference and annotation as two numpy arrays.
+
+    Returns:
+        tuple: A tuple containing two numpy arrays. The first array is the
+        reference volume, and the second array is the annotation volume.
+    """
+    reference_path, annotation_path, volume_resolution = (
+        get_reference_and_annotation_paths(download_dir_path, age, modality)
+    )
+    annotation = load_any(annotation_path)
+    reference = load_any(reference_path)
+    zoom_factors = tuple(volume_resolution / resolution for _ in range(3))
+    reference = zoom(reference, zoom_factors, order=1)
+    if annotation.shape != reference.shape:
+        """
+        we unfortunately did not provide 25um segmentations so
+        we will just downsample the 20um ones
+        """
+        zoom_factors = tuple(
+            ref_dim / ann_dim
+            for ref_dim, ann_dim in zip(reference.shape, annotation.shape)
+        )
+        annotation = zoom(annotation, zoom_factors, order=0)
+    return reference, annotation
+
+
+def retrieve_additional_references(
+    download_dir_path, age, resolution, modalities
+):
+    """This function only needs editing if the atlas has additional reference
+    images. It should return a dictionary that maps the name of each
+    additional reference image to an image stack containing its data.
+    """
+    additional_references = {}
+    for modality in modalities:
+        reference_path, _, volume_resolution = (
+            get_reference_and_annotation_paths(
+                download_dir_path, age, modality
+            )
+        )
+        ref = load_any(reference_path)
+        zoom_factors = tuple(volume_resolution / resolution for _ in range(3))
+        ref = zoom(ref, zoom_factors, order=1)
+        additional_references[modality] = ref
+    return additional_references
+
+
+def retrieve_hemisphere_map():
+    """
+    Retrieve a hemisphere map for the atlas.
+
+    If your atlas is asymmetrical, you may want to use a hemisphere map.
+    This is an array in the same shape as your template,
+    with 0's marking the left hemisphere, and 1's marking the right.
+
+    If your atlas is symmetrical, ignore this function.
+
+    Returns:
+        numpy.array or None: A numpy array representing the hemisphere map,
+        or None if the atlas is symmetrical.
+    """
+    return None
+
+
+def retrieve_structure_information():
+    """
+    Retrieve the structures tree and meshes for the Allen mouse brain atlas.
+
+    Returns:
+        pandas.DataFrame: A DataFrame containing the atlas information.
+    """
+    # Since this atlas inherits from the allen can we not simply get the data
+    # from the bgapi?
```

Comment:
```suggestion
```
Yes! and we do. Nice!

Reply:
Ok potential problem. This atlas actually uses the Allen 2022 segmentations. The Allen in brainglobe uses 2017. Will this cause an issue?

Reply:
I think yes

Reply:
In general ,I think it's probably best not to rely on one atlas to build another. We could get into a situation where one atlas changes, and we're unable to build another.

Reply:
OK, in that case we should fix this before merging.

Reply:
I fixed it :)